prow/config.yaml

```diff
@@ -409,6 +409,19 @@ branch-protection:
           branches:
             master:
               protect: true
+        sail-operator:
+          required_status_checks:
+            contexts:
+            - EasyCLA
```

Comment:
I think this just means "CLA must pass". But we need the CLA to actually run which is owned by LF outside of Istio.

@therealmitchconnors do you know how to get that setup?

Reply:
+1. I'll remove it while we get the CI configured at least for the temporary branch.

Reply:
@therealmitchconnors Do you know how we can get easyCLA installed in istio-ecosystem/sail-operator repo?